### PR TITLE
Cloud economics campaign landing page

### PR DESCRIPTION
**Pull request by caldav**
## Done

- New cloud economics campaign landing page at `/engage/cloud-economics`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/engage/cloud-economics](http://0.0.0.0:8001/engage/cloud-economics)
- Ensure the content matches the [copy doc](https://docs.google.com/document/d/1NtRmAB4Pkphc8WTU8n4-b10knVqqFhMDc_FtGAwQ7nw/edit?usp=sharing)

### CODE REVIEW

**Comment by frankiegranato** [comment]
@caldav 
@anasereijo is going to upload the orange graph graphic that she used on the takeover. I think it might look quite nice on your landing page and would tie-in the landing page to the u.com takeover. Just a suggestion (I don't want to rock the boat - I know you are on a tight deadline for this one!).

**Comment by frankiegranato** [comment]
I am suggesting a change to the wording of the CTA: For the sake on continuity, shall we use the more explanatory "Download the independent report" cta from the u.com takeover. 
cloud economics cta suggestion

**Comment by anasereijo** [comment]
Hi, @caldav can you please try the following suggestions:

- hero banner: would be nice to use the same background image as we have on the homepage to keep consistency. If you do this, please remove the white box around the title
   * background gradient: background-image: linear-gradient(-36deg, #94310F 0%, rgba(233,84,32,0.90) 100%);
   * graph (please anchor to the left side): https://assets.ubuntu.com/v1/13fdbc5b-graph.png
   * logo: https://assets.ubuntu.com/v1/c54da84b-451-Research-NEG.png

- can we make the text 7 col and the download the report 5 col? 


thanks

**Comment by caldav** [comment]
@anasereijo Anchored to the left side gives me [this](https://screenshotscdn.firefoxusercontent.com/images/7821f4bb-70d9-4fcd-823b-2fced770b104.png).
What about [this version](https://screenshotscdn.firefoxusercontent.com/images/8314263d-72d7-4d89-bf05-c9d9acd6ea5e.png) instead? It's much closer to the takevoer, although the lack of CTA and subhead makes the graph more prominent.

**Comment by anasereijo** [comment]
@caldav just one small thing: the gradient should be from dark to light orange and now it is the opposite. Can you please fix that?

File: templates/engage/cloud-economics.html [hunk 1]
```diff
@@ -0,0 +1,401 @@
...
+            </li>
+            <li  class="mktField">
+              <input name="utm_source" class="mktoField  mktoFormCol"  value="" type="hidden">
+            </li>
+            <li  class="mktField">
+              <input name="canonicalUpdatesOptIn" id="canonicalUpdatesOptIn" value="yes" class="mktoField" type="checkbox"><label for="canonicalUpdatesOptIn"></label>
+              <label for="canonicalUpdatesOptIn" class="mktoLabel " >I agree to receive information about Canonical’s products and services.</label>
+            </li>
+            <li  class="mktField">
+              In submitting this form, I confirm that I have read and agree to <a href="https://www.ubuntu.com/legal/dataprivacy" target="_blank" class="mchNoDecorate" id="">Canonical's Privacy Policy</a><input name="RichText" type="hidden">
```
Comment (anthonydillon) [inline templates/engage/cloud-economics.html]: Empty id?

File: templates/engage/cloud-economics.html [hunk 2]
```diff
@@ -0,0 +1,401 @@
...
+            </li>
+            <li  class="mktField">
+              <input name="Consent_to_Processing__c" class="mktoField  mktoFormCol"  value="Yes" type="hidden">
+            </li>
+            <li  class="mktField">
+              <button type="submit" class="mktoButton">Download the independent report</button></span><input name="formid" class="mktoField " value="3219" type="hidden"><input name="lpId" class="mktoField " value="" type="hidden"><input name="subId" class="mktoField " value="" type="hidden"><input name="munchkinId" class="mktoField " value="" type="hidden"><input name="lpurl" class="mktoField " value="" type="hidden"><input name="cr" class="mktoField " value="" type="hidden"><input name="kw" class="mktoField " value="" type="hidden"><input name="q" class="mktoField " value="" type="hidden">
```
Comment (anthonydillon) [inline templates/engage/cloud-economics.html]: Closing span with no opening tag.

**Comment by frankiegranato** [comment]
@caldav I think this page looks FAB

**Comment by caldav** [comment]
Thanks @frankiegranato @anasereijo @anthonydillon , we are still debating the best og:image for social sharing, but happy to get this version merged for now.

**Comment by anasereijo** [comment]
@caldav I'm giving +1 now but please make sure the gradient reflects the one we have on the homepage. thank you

**Pull request merged**